src/components/hotkeys/__tests__/HotkeyLayer.test.js

```diff
@@ -3,19 +3,22 @@ import sinon from 'sinon';
 
 import HotkeyRecord from '../HotkeyRecord';
 import HotkeyLayer from '../HotkeyLayer';
+import HotkeyService from '../HotkeyService';
 
 const sandbox = sinon.sandbox.create();
 
+jest.mock('../HotkeyService');
+
 describe('components/hotkeys/HotkeyLayer', () => {
     beforeEach(() => {
         // This is required to prevent actually invoking HotkeyService, which causes
         // HotkeyService tests to fail
-        HotkeyLayer.__Rewire__('HotkeyService', class {});
+        HotkeyService.mockImplementation(() => {});
```

Comment:
Is it necessary to mock every time? Can you mock once on the top then `mockReset` before or after each?

Reply:
probably not. I just did a direct translation of the existing tests. I'll see if that works